tests/test_interpolation.py

```diff
@@ -126,8 +126,9 @@ def test_precomputed_interpolation_time():
     op = Operator(eqn)
     op(time_m=0, time_M=4)
 
+    # NOTE: Work out why this change was needed before any merge
```

Comment:
?

Reply:
I remember I needed to change this assert as it started to break. Could have been a python update and the same change was also needed in master?